rs3_trading/database/ge_tick_database.py

```diff
@@ -1,17 +1,26 @@
-import duckdb
+def create_price_table(con):
+    con.execute('create table if not exists rs3_price_data(time datetime, item varchar, item_name varchar, price long, primary key(item, time))')
 
-con = duckdb.connect('GETest')
 
+def create_volume_table(con):
+    con.execute('create table if not exists rs3_volume_data(time datetime, item varchar, item_name varchar, volume long, primary key(item, time))')
 
-def create_ge_tick_table(con):
-    con.execute('create table if not exists ge_tick_data(time datetime, item varchar, item_name varchar, price long)')
 
-
-def insert_into_tick_table(con, data):
-    con.execute("""insert into ge_tick_data(time, item, item_name, price)
+def insert_into_price_table(con, ge_tick_dataframe):
```

Comment:
Make a method called insert_table that takes the table_name, column_list and dataframe to insert. Make both the insertion methods use this function